sdk/extensions/Microsoft.Extensions.Azure/src/Internal/ClientFactory.cs

```diff
@@ -93,6 +94,15 @@ internal static TokenCredential CreateCredential(IConfiguration configuration, T
             var certificate = configuration["clientCertificate"];
             var certificateStoreName = configuration["clientCertificateStoreName"];
             var certificateStoreLocation = configuration["clientCertificateStoreLocation"];
+            var additionallyAllowedTenants = configuration["additionallyAllowedTenants"];
```

Comment:
One other thought - should we make this a JSON list configuration rather than delimited string?

Reply:
I'm not sure if that makes it any easier for the user. Plus, then we'd also have to handle invalid json.

Reply:
I agree I think the semi-colon delimited string is less error prone. This is the format expected in the environment variable `AZURE_ADDITIONALLY_ALLOWED_TENANTS`, and I think semi-colon delimited is pretty standard for environment variables. But if JSON is more regularly used in functions config, we could consider using it here.

Reply:
woops, it is currently comma delimited - should it be semi-colon?